backend/Modules/Users/HitMeApp.Users/Exceptions/UserAlreadyExistsException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace HitMeApp.Users.Exceptions
+{
+    internal class UserAlreadyExistsException : Exception
```

Comment:
All infrastructure related exceptions should inherit `ApplicationException` provided in https://github.com/Somsiady-Inc/HitMeApp/pull/15 when it's merged. 

That would not only make exceptions more consistent, but will be really important for some coding magic later on.

Reply:
Done